src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlConnectionWrapperFactory.cs

```diff
@@ -4,7 +4,7 @@
 // -------------------------------------------------------------------------------------------------
 
 using EnsureThat;
-using Microsoft.Health.Fhir.SqlServer.Configs;
+using Microsoft.Health.SqlServer.Configs;
```

Comment:
Shouldn't these move into the common code too?

Reply:
So far this is only a concept that's needed within the fhir-server. We can move it, but that would require a bit more refactoring as the common transaction interfaces live in the fhir-server core.

Reply:
The DICOM code actually has these registered but that might be because I just copied the code without looking at it closely. Let's not move the code and we can refactor the code in DICOM side when we incorporate the nuget package.